projects/make_project.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
```

Comment:
So this script actually is included functionality in `brew barkly`:

https://github.com/barklyprotects/homebrew-barkly#create
https://github.com/barklyprotects/homebrew-barkly/blob/master/lib/create.sh

This may be simpler in terms of having a template rather than having it inline shell, thoughts?